patch-bms-novhs/src/main.rs

```diff
@@ -47,23 +52,41 @@ fn run() -> Result<()> {
 
     let bms_path = find_bms(args.input)?;
     let mut map = open_bms(&bms_path)?;
-    ensure_bms(&map)?;
+    let bms_version = find_bms_version(&map)?;
 
-    const REPLACEMENT_NOP: &[u8] = &[0x0f, 0x1f, 0x44, 0x00, 0x00]; // lea eax, eax * 1 + 0
+    const REPLACEMENT_NOP: &[u8] = &[0x0F, 0x1F, 0x44, 0x00, 0x00]; // lea eax, eax * 1 + 0
 
-    let patches = vec![
+    let patches_435u1 = vec![
         Patch {
-            offset: 0x00022544,
-            original: &[0xe8, 0x87, 0x55, 0x00, 0x00],
+            offset: 0x0002_2544,
+            original: &[0xE8, 0x87, 0x55, 0x00, 0x00],
```

Comment:
I converted the hex to ASCII uppercase as God intended. If you feel strongly about this I will revert.